README.md

```diff
@@ -26,39 +42,31 @@ pip install -r requirements.txt
-* [Swagger github pages][7] show this alternate version to grow.
+* [Swagger github pages][6] show this alternate version to grow.
 * `API.yaml` should gradually become the wanted version of the TSP, as this alternate one shows the reference TSP.
 * Such reference and future (forecasted) TSP versions will be documented accordingly as these yaml files evolve.
 
-1. Import all [TC][2] and [incubator][3] projects in Eclipse; branch, Target Platform and API Baseline set to `master`.
-1. Run trace-server or project [1] below as Eclipse Application configured this way (or similar):
-   * `-Xms512m -Xmx1g`.
-   * Launch with `all workspace and enabled target plug-ins`.
-1. Browse [to here][4] ([swagger][5]) or so to generate server's TSP.
-1. Bring the resulting file over; e.g.: `mv ~/Downloads/openapi.yaml .`
-1. Update the latter with its license information and remove extra information: `./openapi.py`
-1. The resulting git diff may then be pushed for review, at will.
-
-[1] org.eclipse.tracecompass.incubator.trace.server.jersey.rest.core
-
-[2]: https://projects.eclipse.org/projects/tools.tracecompass/developer
-[3]: https://projects.eclipse.org/projects/tools.tracecompass.incubator/developer
-[4]: http://localhost:8080/tsp/api/openapi.yaml
-[5]: https://github.com/swagger-api/swagger-core/wiki/Swagger-2.X---Integration-and-configuration#openapiresource
-[6]: https://theia-ide.github.io/trace-server-protocol/
-[7]: https://theia-ide.github.io/trace-server-protocol/swagger/
-[8]: https://marketplace.visualstudio.com/items?itemName=42Crunch.vscode-openapi
+1. Import all [TC][1] and [incubator][2] projects in Eclipse; branch, Target Platform and API Baseline set to `master`.
+2. Open `traceserver.product` file in plug-in `org.eclipse.tracecompass.incubator.trace.server.product`
+3. Click on the run button on the top right corner of the opened `traceserver.product`
+4. Browse [to here][3] ([swagger][4]) or so to generate server's TSP.
+5. Bring the resulting file over; e.g.: `mv ~/Downloads/openapi.yaml .`
+6. Update the latter with its license information and remove extra information: `./openapi.py`
+7. The resulting git diff may then be pushed for review, at will.
+
+[1]: https://projects.eclipse.org/projects/tools.tracecompass/developer
+[2]: https://projects.eclipse.org/projects/tools.tracecompass.incubator/developer
+[3]: http://localhost:8080/tsp/api/openapi.yaml
+[4]: https://github.com/swagger-api/swagger-core/wiki/Swagger-2.X---Integration-and-configuration#openapiresource
+[5]: https://theia-ide.github.io/trace-server-protocol/
+[6]: https://theia-ide.github.io/trace-server-protocol/swagger/
+[7]: https://marketplace.visualstudio.com/items?itemName=42Crunch.vscode-openapi
+[tspClient]: https://github.com/theia-ide/tsp-typescript-client
+[tcServer]: https://download.eclipse.org/tracecompass.incubator/trace-server/rcp/
+[apiProposed]: https://theia-ide.github.io/trace-server-protocol/proposed/
```

Comment:
This will be added in a separate PR, because it will needs to be done on the `gh-pages` branch.

Reply:
Remove `openapi.yaml`, now that it is being replaced with `API.yaml`?

Reply:
Also, these (many) diffs look like being more than just the act of splitting the yaml files or APIs. I find this sole commit hard to review. Can we split these into more than just one commit, so we can review by iterating through each scoped change? The commits can stay in this same PR no problem I think. -Thanks.

Reply:
> Remove `openapi.yaml`, now that it is being replaced with `API.yaml`?

Makes sense!

Reply:
> Also, these (many) diffs look like being more than just the act of splitting the yaml files or APIs. I find this sole commit hard to review. Can we split these into more than just one commit, so we can review by iterating through each scoped change? The commits can stay in this same PR no problem I think. -Thanks.

Well, the diffs are mainly because the original references were numbered and I had remove some and re-number the references. Beside different numbers, the text is pretty much the same. Unfortunately, github diff shows to many changes which are not really changes.  I can split it multiple commits, but I don't really see it's worth because the changes are due to the update.